source/jormungandr/jormungandr/scenarios/ridesharing/ridesharing_service.py

```diff
@@ -33,6 +33,9 @@
 import six
 import logging
 from jormungandr import utils
+from collections import namedtuple
+
+Rs_FeedPublisher = namedtuple('Rs_FeedPublisher', ['id', 'name', 'license', 'url'])
```

Comment:
It seems to be over-engineered, if you have better solutions please tell me

Reply:
Looks fine, but please choose your case 👀

Reply:
@kinnou02 because this one is hashable :)